whylog/assistant/regex_assistant/regex.py

```diff
@@ -0,0 +1,119 @@
+"""
+Regex verification and creating (but not finding groups in regex)
+"""
+import re
+
+special_characters = frozenset(['.', '^', '$', '*', '+', '{', '}', '?', '[', ']', '|', '(', ')'])
+
+
+class NoDateGroupError(Exception):
```

Comment:
In my opinion all exceptions of whylog should be on whylog/exceptions file and should inherit from WhylogException.